src/Components/WebAssembly/WebAssembly.Authentication/src/Options/OidcProviderOptions.cs

```diff
@@ -59,8 +59,11 @@ public class OidcProviderOptions
         public string ResponseMode { get; set; }
 
         /// <summary>
-        /// Gets or sets the extra query parameters to use on the authorization flow. 
+        /// Gets or sets the additional provider parameters to use on the authorization flow. 
         /// </summary>
-        public IDictionary<string, string> ExtraQueryParams { get; } = new Dictionary<string, string>();
+        /// <remarks>
+        /// These parameters are for the IdP and not for the application. Using those parameters in the application in any way on the login callback will likely introduce security issues as they should be treated as untrusted input.
+        /// </remarks>
+        public IDictionary<string, string> AdditionalProviderParameters { get; } = new Dictionary<string, string>();
```

Comment:
```suggestion
        [JsonPropertyName("extraQueryParams")]
        public IDictionary<string, string> AdditionalProviderParameters { get; } = new Dictionary<string, string>();
```

Reply:
All other json property names are using underscores and lowercase. Should this be `extra_query_params` to be consistent or is this intentional?

Reply:
This is intentional to match the UserManagerSettings object of oidc-client JS shape.

Reply:
@javiercn would you please re-review this PR?